build.gradle

```diff
@@ -14,6 +14,7 @@ buildscript {
         junitVersion = '4.13'
         androidTestVersion = '1.2.0'
         androidEspressoVersion = '3.2.0'
+        ndkVersion = '21.0.6113669'
```

Comment:
Why do we need to specify ndkVersion btw?

Reply:
I see the warning message that `ANDROID_NDK_HOME` is deprecated now. Any suggestion?

Reply:
I don't think we are using that env?

Reply:
Otherwise, I cannot get the project built with Android Studio 4.0...